modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+
+> Given the five changes in the Unreleased section of the Advanced
+> Typography plugin’s [changelog][1], is it sensible to assume this plugin
+> is getting a version bump?
+
+[0]: <https://notion.so/TinyMCE-6-4-Bill-of-materials-BOM-885a34e3b9884be0b69cd30eba7c618b?d=5267c02d507140c2882a18c2a9c7c753#0067a51f89bc45f1a99cc1b80a6e7a39>
+
+[1]: <https://github.com/tinymce/typography/blob/main/CHANGELOG.md>
+////
+
+
+
+=== Checklist 2.0.4
+
+The {productname} 6.4 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.3 includes the following fix
+
+////
+To quote my [comment, left on the TinyMCE 6.4 BoM][2] in Notion
+
+> What changes were made to the Checklist plugin? The [changelog][3]
+> contains no entries beyond 2022-11-23 and the only [commits this
+> year][4] have been dependabot bumps or back-end changes.
+
+[2]: <https://notion.so/TinyMCE-6-4-Bill-of-materials-BOM-885a34e3b9884be0b69cd30eba7c618b?d=a1149ea94c2e459b80773fa2e59a6cc6#e826f887f0654bafb3ec0fdcb1659385>
+
+[3]: <https://github.com/tinymce/santa/blob/main/CHANGELOG.md>
+
+[4]: <https://github.com/tinymce/santa/commits/main>
+////
+
+For information on the **Checklist** plugin, see: xref:checklist.adoc[Checklist].
+
+
+
+=== Merge Tags 1.2.0
+
+The {productname} 6.4 release includes an accompanying release of the **Merge Tags** premium plugin.
+
+**Merge Tags** 1.2.0 includes the following change.
+
+==== The mceMergeTag command now preserves the selected content as a plain text and wraps it in a merge tag prefix and suffix if the content length exceeds the limit.
+
+// https://ephocks.atlassian.net/browse/TINY-9444
+
+In previous versions of {productname}, `mceMergeTag's` would not preserve the selected content within the merge tag `prefix` and `suffix` in some events.
+
+As a consequence, when the user applied the trailing suffix `}}` to the end of the content, the selected content wrapped within the `mceMergeTag` would not be preserved as plain text.  This would occur specifically in instances when the `mceMergeTag` content length exceeded the character limit set by {productname}.
+
+To fix this issue, improvements to the `mceMergeTag` commands were introduced with the release of {productname} 6.4, that focused on adding a new feature that impacts the way text_patterns handle specific cases such as when `mceMergeTag's` exceed the character limit set by {productname}. These improvements now wrap any content that exceeds the `mceMergeTag` limit, by converting the content into *plain text* whilst encapsulating it in the `prefix` and `suffix`.
+
+As a result of these improvements, the `mceMergeTag` command now preserves the selected content as a plain text and wraps it in a merge tag `prefix` and `suffix` as expected.
+
+[NOTE]
+Any content that is covered into plain text that exceeds the `mceMergeTag` character limit rules, will not be converted into an official `mceMergeTag`.
```

Comment:
```suggestion
If any content is embraced with a merge tag prefix and suffix but exceeds the character limit for merge tag content length, it will not be converted into an official merge tag.
```